keyboards/kbdfans/kbd67/mkiirgb/v1/config.h

```diff
@@ -1,10 +1,10 @@
 #pragma once
 #include "config_common.h"
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x4b42 // KB
```

Comment:
The hex value should be uppercase.

```suggestion
#define VENDOR_ID       0x4B42 // KB
```